kview/src/main/java/dev/ikm/komet/kview/mvvm/view/descriptionname/DescriptionNameController.java

```diff
@@ -133,23 +102,38 @@ public DescriptionNameController() { }
     @FXML
     public void initialize() {
         clearView();
+        populateDialectComboBoxes();
 
-        descrNameViewModel
-                .setPropertyValue(STATUS, TinkarTerm.ACTIVE_STATE);
+        //TODO These are temp hard coded values:
+        // Should use
+        // setupComboBox(nameDescriptionType, fetchDescendentsOfConcept(getViewProperties(), TinkarTerm.DESCRIPTION_TYPE.publicId()));
 
-        populateDialectComboBoxes();
+        setupComboBox(nameDescriptionType, DESCRIPTION_TYPE_OPTIONS); // Hard coded
```

Comment:
Can this call the newly created function? Assuming a function exists from the earlier comment.

Reply:
Done